_includes/sidebar.html

```diff
@@ -89,32 +93,38 @@
                     return "stable";
                   })();
 
-                  function renderItems(items, tier) {
+                  function renderItems(items, paths) {
                     if (!items || items.length == 0)
                         return $();
 
                     var lis = items.map(function (item) {
                       var urls = (item.urls || []).map(function (url) {
                           return baseUrl + url.replace("${VERSION}", pageVersion);
                       });
-                      var subitems = renderItems(item.items, tier + 1);
+                      var active = (urls.indexOf(location.pathname) !== -1);
+                      if (active) {
+                        $("#ch-breadcrumbs")
```

Comment:
I think I see what's happening here, you want to eventually overwrite with the value that occurs at the deepest point in the recursion? After figuring this out I think this logic is subtle enough that it could do with a comment explaining what's happening here, or maybe even pull this whole thing out into a separate function?